jiant/preprocess.py

```diff
@@ -235,18 +235,20 @@ def _build_vocab(args, tasks, vocab_path: str):
 
 def build_indexers(args):
     indexers = {}
-    if not args.input_module.startswith("bert") and args.input_module not in ["elmo", "gpt"]:
+    if args.input_module in ["scratch", "glove", "fastText"]:
         indexers["words"] = SingleIdTokenIndexer()
-    if args.input_module == "elmo":
+    elif args.input_module == "elmo":
```

Comment:
is `elmo-chars-only` handled properly here?

Reply:
Good catch. Not a new bug, but I'll fix it and quickly test elmo-chars-only.